### DISCUSSION TITLE
Friendly point of information: Using `wget` does not install distant.

### DISCUSSION THREAD

**Discussion by anoduck** [chipsenkbeil/distant#230] [General]
Just an annoying, yet friendly, point of information. Running `wget https://sh.distant.dev | sh` only fetches `index.html`. It does not install Distant, but `curl -L https://sh.distant.dev | sh` does.

It is a miniscual issue, barely worth noting.

**Comment by chipsenkbeil** [comment]
Ah, you're right! It should be `wget -q -O- sh.distant.dev | sh`, I believe.

**Reply by anoduck** [reply]
Yep, that works. As said, it is a very insignificant issue.

**Comment by chipsenkbeil** [comment]
Updated the website and the README.
